packages/snaps-cli/src/commands/build/build.e2e.test.ts

```diff
@@ -2,28 +2,13 @@ import { promises as fs } from 'fs';
 import { join } from 'path';
 
 import type { TestRunner } from '../../test-utils';
-import { getCommandRunner, SNAP_BROWSERIFY_DIR } from '../../test-utils';
+import { getCommandRunner } from '../../test-utils';
 
 describe('mm-snap build', () => {
   let runner: TestRunner;
-  let originalManifest: string;
-
-  beforeEach(async () => {
-    // Since this is an end-to-end test, and we're working with a "real" snap,
-    // we have to make a copy of the original snap manifest, so we can modify it
-    // and reset it after the test.
-    originalManifest = await fs.readFile(
```

Comment:
Why do we not need this anymore?

Reply:
This was only needed for Browserify, I think because the shasum changed in some cases in the tests? Running the E2E test for Webpack doesn't modify the manifest.